feelslikemonday/app/src/main/java/com/example/feelslikemonday/DAO/VoidCallback.java

```diff
@@ -0,0 +1,5 @@
+package com.example.feelslikemonday.DAO;
+
+public interface VoidCallback {
```

Comment:
any particular reason for why VoidCallback and UserCallback two different interfaces? I thought we were just using a general callback for whenever a userDAO function is complete.

Reply:
Different method signatures being used